docs/2_step_by_step/2_3_login_to_jina_ecosystem.md

```diff
@@ -1,3 +1,4 @@
+(login-to-jina-ecosystem)=
 # Login to Jina ecosystem
 
 Since Finetuner leverages cloud resources for fine-tuning,
```

Comment:
should we use `Finetuner` here?

Reply:
we always use the up case Finetuner @jemmyshin